lib/awesome_spawn/command_result_error.rb

```diff
@@ -1,9 +1,13 @@
 module AwesomeSpawn
   class CommandResultError < StandardError
+    include AwesomeSpawn::Logging
+
     # @return [CommandResult] The command that caused the error
     attr_reader :result
 
     def initialize(message, result)
+      logger.error("AwesomeSpawn: #{message}")
+      logger.error("AwesomeSpawn: #{result.error}")
```

Comment:
I'm debating this.  I get why it was put here, but I feel like it really belongs in the caller.  Are there a lot of callers that create `CommandResultError` objects?